changelog/fix-e2e-analytics

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: Fix only relates to E2E tests.
```

Comment:
@dmallory42 Just noting that this is not exactly a fix for failing E2E tests. As the issue mentioned, it's present on a new WooCommerce store with the latest `develop` of WooCommerce Payments, even without on-boarding.

Reply:
Good point. I didn't mention the error because thankfully the E2E tests caught it before it was released, so it was never customer-facing. However I can add it to the changelog if you think it is still worth mentioning?

Reply:
I think it's worth mentioning because devs can quickly check what the change is about :)

Reply:
Updated in b1351fba. :)